### PR TITLE
add tests to prove fix for #627

### PR DESCRIPTION
**Pull request by wakingrufus**

### CODE REVIEW

File: plugin/src/test/kotlin/org/jlleitschuh/gradle/ktlint/DisabledRulesTest.kt [hunk 1]
```diff
@@ -63,17 +63,29 @@ class DisabledRulesTest : AbstractPluginTest() {
...
+                } else {
+                    editorConfig.appendText(
+                        """
+                        root = true
+
+                        [*.kt]
+                        disabled_rules = final-newline
+                        """.trimIndent()
+                    )
+                }
```
Comment (stkent) [inline plugin/src/test/kotlin/org/jlleitschuh/gradle/ktlint/DisabledRulesTest.kt:88]: Suggestion: add one more block for the renamed property in ktlint 0.47:

https://github.com/pinterest/ktlint/blob/master/CHANGELOG.md#editorconfig-property-disabled_rules

> #### .editorconfig property "disabled_rules"
> 
> The `.editorconfig` property `disabled_rules` (api property `DefaultEditorConfigProperties.disabledRulesProperty`) has been deprecated and will be removed in a future version. Use `ktlint_disabled_rules` (api property `DefaultEditorConfigProperties.ktlintDisabledRulesProperty`) instead as it more clearly identifies that ktlint is the owner of the property. This property is to be renamed in `.editorconfig` files and `ExperimentalParams.editorConfigOverride`.   
> 
> Although, Ktlint 0.47.0 falls back on property `disabled_rules` whenever `ktlint_disabled_rules` is not found, this result in a warning message being printed.

```suggestion
                if (SemVer.parse(ktLintVersion) >= SemVer.parse("0.48.0")) {
                    // new way of disabling rules introduced in 0.48
                    editorConfig.appendText(
                        """
                        root = true

                        [*.kt]
                        ktlint_standard_final-newline = disabled
                        """.trimIndent()
                    )
                } else if (SemVer.parse(ktLintVersion) >= SemVer.parse("0.47.0")) {
                    // disabled rules property renamed in 0.47
                    editorConfig.appendText(
                        """
                        root = true

                        [*.kt]
                        ktlint_disabled_rules = final-newline
                        """.trimIndent()
                    )
                } else {
                    editorConfig.appendText(
                        """
                        root = true

                        [*.kt]
                        disabled_rules = final-newline
                        """.trimIndent()
                    )
                }
```

File: plugin/src/test/kotlin/org/jlleitschuh/gradle/ktlint/DisabledRulesTest.kt [hunk 2]
```diff
@@ -89,6 +110,7 @@ class DisabledRulesTest : AbstractPluginTest() {
                 build(CHECK_PARENT_TASK_NAME) {
                     assertThat(task(":$mainSourceSetCheckTaskName")?.outcome).isEqualTo(TaskOutcome.SUCCESS)
                     assertThat(output).doesNotContain("Property 'ktlint_disabled_rules' is deprecated")
+                    assertThat(output).doesNotContain("Property 'disabled_rules' is deprecated")
```
Comment (stkent) [inline plugin/src/test/kotlin/org/jlleitschuh/gradle/ktlint/DisabledRulesTest.kt:113]: 🔥

**Review by stkent — APPROVED** [review]
💯 Updates look super, thank you!

**Review by JLLeitschuh — APPROVED** [review]
LGTM!

**Pull request merged**